jvm/src/test/scala/tastyquery/TypeSuite.scala

```diff
@@ -389,6 +398,36 @@ class TypeSuite extends BaseUnpicklingSuite(withClasses = true, withStdLib = tru
         clues(A.bounds)
       )
     }
+
+    extension (tpe: Type)
+      def isRawGenericJavaClass(using BaseContext): Boolean =
+        tpe.isApplied(_.isRef(GenericJavaClass), List(_.isBounded(_.isRef(NothingClass), _.isRef(AnyClass))))
```

Comment:
It would probably be worth having helpers `isAny` and `isNothing` on `Type`.

Reply:
done